src/main/content/_assets/js/builds.js

```diff
@@ -266,58 +309,74 @@ function render_builds(builds, parent) {
-                    var num_packages = package_locations.length;
-                    var version_column = $(
+
+                    // Assume the .sig files from the allowed builds are also allowed
+                    // without checking the .sig files.
+                    beta_package_locations = 
+                        getAllowedBuilds('runtime_betas', beta_package_locations, version);
+                    
+                    var num_beta_packages = beta_package_locations.length;
+                    var beta_version_column = $(
                         '<td headers="' +
                             tableID +
                             '_version" rowspan="' +
-                            num_packages +
+                            num_beta_packages +
                             '">' +
                             build.version +
                             '</td>'
                     );
-
-                    for (var i = 0; i < package_locations.length; i++) {
-                        var row = $('<tr></tr>'); // create a new row for each item in package_locations
-                        var package_name = package_locations[i]
+            
+                    for (var d = 0; d < beta_package_locations.length; d++) {
+                        // create a new row for each item in package_locations
+                        var beta_row = $('<tr></tr>');
+                        //========== Get URL for the .zip file
+                        var beta_package_name = beta_package_locations[d]
                             .split('=')[0]
                             .toLowerCase();
-                        var href = package_locations[i].split('=')[1];
-                        var download_column = $(
-                            '<td headers="' +
-                                tableID +
-                                '_download"><a href="' +
-                                href +
-                                '" class="' +
-                                analytics_class_name +
-                                '" rel="noopener">' +
-                                download_arrow +
-                                'ZIP</a></td>'
+                        var beta_zip_href = beta_package_locations[d].split('=')[1];
+                        //========== Get URL for the .sig file corresponding to the .zip file
+                        // The name has a lot of dots, so have to use lastIndexOf to separate
+                        // filename from file extension
+                        var beta_sig_name = beta_package_name.substring(0, beta_package_name.lastIndexOf('.'));
+                        var beta_sig_href = 
+                            getURLForSig(beta_package_sig_locs, beta_sig_name+'.sig');
+                        //========== Get URL for the .sha2 file
+                        var beta_sha2_href = ''; // TODO: Surface the href when DHE API has this data
```

Comment:
Do we have an issue we can mark here for the TODO?

Reply:
I opened not to put the issue number in the comment.  The issue is 
 https://github.com/OpenLiberty/openliberty.io/issues/1734